airbyte-ci/connectors/pipelines/README.md

```diff
@@ -398,6 +398,7 @@ This command runs the Python tests for a airbyte-ci poetry package.
 ## Changelog
 | Version | PR                                                         | Description                                                                                               |
 | ------- | ---------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+|         |                                                            | There's a lotta open PRs right now. Lol  
```

Comment:
❓

Reply:
I was waiting for approval since there were so many open PRs I knew I'd have to change this anyway 😄 it was a reminder to bump the version in pyproject.toml